packages/components/namespace-store/namespace-store-form.tsx

```diff
@@ -85,6 +85,11 @@ const NamespaceStoreForm: React.FC<NamespaceStoreFormProps> = withHandlePromise<
     redirectHandler,
     namespace
   } = props;
+  const getFilterResource = React.useCallback(
+    (pvcObj: PersistentVolumeClaimKind) =>
+      pvcObj?.status?.phase === "Bound" &&
+      pvcObj?.spec?.accessModes.some((mode) => mode === "ReadWriteMany"), []
+  );
```

Comment:
on second thought, we can move this function out of React FC itself. That will also work and will be even better than using useCallBack...